assets/js/hale-scripts.js

```diff
@@ -0,0 +1,14 @@
+/*
+	hale script v1.0.0
+*/
+
+/*
+	Search button in header - show-hide the search field
+*/
+document.getElementById("search-show-hide").addEventListener('click', 
```

Comment:
Do we need one for the hamburger menu too? (Sorry if that's been done in another PR!)

Reply:
The menu one is dealt with by the GDS JavaScript already.